app/templates/home.html

```diff
@@ -1 +1,14 @@
-<h1>Home Page</h1>
\ No newline at end of file
+<div class = 'container'>
+
+  <section>
+    <div class="time"><h1>{{ home.timer.counter | timecode }}</h1></div>
+    <div ng-show="!home.timer.isTimeRunning">
+      <button ng-click="home.timer.startSession()" class='btn btn-primary'>Let's Get Working</button>
```

Comment:
There may be a way to have only one button in the markup and change its functionality and text based on the status of the timer.